client/table.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter@cockroachlabs.com)
+
+package client
+
+import (
+	"log"
+
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/proto"
+)
+
+// CreateTable ...
+func (db *DB) CreateTable(name string, schema proto.TableSchema) error {
+	return nil
+}
+
+// RowBuilder ...
+type RowBuilder struct {
+	tableName string
+	columns   []string
```

Comment:
The `r.Rows({"id": 9})` syntax isn't feasible unless there is a feature in Go I don't know about. The closest would be something like `r.Rows(client.Cols{"id": 9})`.

An alternative to these approaches would be to consider using struct tags in some way:

```
type User struct {
  ID int `db:"id"`
  Name string `db:"name"`
}

u := &User{ID: 1}
if err := db.BindStruct("users", u); err != nil {
  ...
}
if err := db.GetStruct(u); err != nil {
  ...
}
```